channels/alpha

```diff
@@ -7,6 +7,8 @@ spec:
     - name: kope.io/k8s-1.5-debian-jessie-amd64-hvm-ebs-2017-01-09
       providerID: aws
       kubernetesVersion: ">=1.5.0"
+    - providerID: gce
+      name: "cos-cloud/cos-stable-56-9000-84-2"
```

Comment:
I wasn't able to pull the COS images up with just `gcloud compute images list` but I was able to find them via:

```
gcloud compute images list \
    --project cos-cloud \
    --no-standard-images
```